### ISSUE TITLE
Bug: Search bar for filter clips off screen 

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/posthog#11293]
## Bug description

Trying to make a filter on my Macbook Air. I want to filter by `initial referring domain` as part of a query @mariusandra had and...wait, where's the search field for properties? This is full screen, with me scrolled up to the top of the window, but...

<img width="1440" alt="Screenshot 2022-08-15 at 11 44 21" src="https://user-images.githubusercontent.com/84011561/184622311-d082f771-eefb-4a95-9dcd-217e12bfd2cf.png">

Turns out I have to scroll down a little bit to get it visible, as that forces the modal to a better position...

<img width="1440" alt="Screenshot 2022-08-15 at 11 44 36" src="https://user-images.githubusercontent.com/84011561/184622425-daf73c75-3699-449e-ac58-f51009a1fb00.png">

## How to reproduce

1.
2.
3.

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: please provide

## Additional context

Tagging @clarkus and @lottiecoxon ~for shits and giggle~ as it's a UI issue.

**Comment by spawan316** [comment]
I could not reporoduce the bug in my machine.
Heres the loom video for this: 
https://www.loom.com/share/f4828025d6444bbf936c2ba9eb693b1b?fbclid=IwAR2R8SJtcHC4wynbj-SIWyNXgx-2t2B3oFd9OZmSsBCOtyW0VZM_YGTbZps

**Comment by joethreepwood** [comment]
Yep, I can't reproduce this either for now - so closing. Thanks!
